app/shared/src/commonTest/kotlin/ui/subject/episode/video/PlayerFloatingTipsStateTest.kt

```diff
@@ -0,0 +1,159 @@
+package ui.subject.episode.video
+
+import me.him188.ani.app.ui.subject.episode.video.PlayerFloatingTipsState
+import me.him188.ani.app.videoplayer.ui.state.Chapter
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class PlayerFloatingTipsStateTest {
+    val _24minute = 24 * 60 * 1000L
+    val _12minute = 12 * 60 * 1000L
+
+    val chapters = listOf(
+        Chapter("op", 90_000L, 10_000L),
+        Chapter("chapter1", 5000L, 115_000L),
+        Chapter("op 60s", 60_000L, 10_000L),
+    )
+
+    @Test
+    fun `pos before op 10s when 12 minute`() {
```

Comment:
```suggestion
    @Test
    fun `12 minutes - pos before op 10s`() {
```
我觉得这种命名可能比较容易分辨

这样我能清晰看到你为 `12 minutes` 类型写了几个测试